src/lib/cache/subCache.cpp

```diff
@@ -33,6 +33,7 @@
 
```

Comment:
Although there shouldn't be any functional change with regard this PR, I think a CNR entry should be included, due to:

1. It may have a potential impact (let's hope that benefitial) in performance
2. The change is critical enougth to appear in future release notes. This is the kind of things that could be tricky to realize when unknown problems occur :)

Something like this:

```
Hardening: csub cache implemenetation now uses hashmap instead of linked list, thus potentially improving search performance
```

In addition, I understand this PR could be the solution for #2216. In that case, another additional entry (of kind "Fix") should be also added.